evm-contracts/src/v0.6/VRFConsumerBase.sol

```diff
@@ -146,4 +147,12 @@ abstract contract VRFConsumerBase is VRFRequestIDBase {
     vrfCoordinator = _vrfCoordinator;
     LINK = LinkTokenInterface(_link);
   }
+
+  // fulfillRandomness_ is called by VRFCoordinator when it receives a valid VRF
```

Comment:
fix comment

Reply:
fixed

Reply:
But it's called `rawFulfillRandomness` now? :)